cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -326,10 +326,11 @@ def test_block_with_children(self):
         self.assertEqual(course_child_block.data, data_value)
         self.assertEqual(course_child_block.display_name, name_value)
 
-    def test_change_after_first_sync(self):
```

Comment:
Renamed this test, and beefed it up with assertions from test_refresh_fails_for_unknown_library.